Doc/library/asyncio.rst

```diff
@@ -4,7 +4,7 @@
 .. module:: asyncio
    :synopsis: Asynchronous I/O.
 
---------------
+-------------------------------
```

Comment:
The number of lines in the header is supposed to be equal to the number of characters in the most recent non-empty line. I'm not sure if it makes a difference in rendering, but the doc tests will often have errors if it doesn't line up exactly.

Reply:
This is weird, because it's a line separator and not a header.  Could you please double check that a horizontal line is rendered and not something weird is happening instead?

Reply:
> This is weird, because it's a line separator and not a header

Oops, I meant line separator. I was thinking of headers because I was just reading over a previous comment.

>  Could you please double check that a horizontal line is rendered and not something weird is happening instead?

Here's what the current Sphinx rendering looks like:
![image](https://user-images.githubusercontent.com/44193521/66612244-2d0a2100-eb8f-11e9-9660-44ca3c7354fc.png)

Which is the same as the current rendering (https://docs.python.org/3/library/asyncio.html):
![image](https://user-images.githubusercontent.com/44193521/66612277-41e6b480-eb8f-11e9-9ccc-34681af0b478.png)

There's no external change made, it's just an reST convention (IIRC). I can remove it if needed though.

Reply:
From the reST docs (http://docutils.sourceforge.net/docs/ref/rst/restructuredtext.html#sections):

" An underline/overline is a single repeated punctuation character that begins in column 1 and forms a line extending at least as far as the right edge of the title text."

"A blank line after a title is optional. "

So the separators are considered "Section Titles" regardless of whether or not there's a blank line in between them, and the repeated punctuation character (in this case `-`) is supposed to match the right edge of the title text.